src/resources/nfts/simplehash/index.ts

```diff
@@ -65,7 +65,12 @@ export async function fetchSimpleHashNFTListing(
   // array of all eth listings on OpenSea for this token
   let listings: SimpleHashListing[] = [];
   let cursor = START_CURSOR;
-  const chain = getSimpleHashChainFromNetwork(network);
+  const chain = RainbowNetworks.find(n => n.value === network)?.nfts?.simplehashNetwork;
```

Comment:
i think this only exists for zora and mainnet

Reply:
prob just need to map by simplehashNetwork || value or smthn 🤔

Reply:
oh youre right. any harm in adding them for all networks?

Reply:
nah im coo with it just make the type required